### DISCUSSION TITLE
Diagnostic pop-up covering code

### DISCUSSION THREAD

**Discussion by kemelzaidan** [helix-editor/helix#11096] [Q&A] [answered]
![image](https://github.com/helix-editor/helix/assets/897436/f7c522e0-a261-4049-8947-8d15b8762c23)
I don't know if it's clear on the image above, but the lsp dialog box above is covering the source code while I'm typing it. But since I haven't finished typing yet, it shows as an error. Is there a way to close it, or turn it on and off when I'm debugging it?

Thanks,

Kemel Zaidan

**Comment by matta** [comment, marked as answer]
This issue is tracked in issue https://github.com/helix-editor/helix/issues/1462.

 I was able to reproduce it in Rust source by editing code at the very top of the file within a narrow window:

![Screenshot from 2024-07-05 08-35-31](https://github.com/helix-editor/helix/assets/37314/e0814b62-e0fc-43de-a067-be44cf359343)

This suggests a workaround: use a wider terminal window, or a much taller one, perhaps only temporarily. Not very satisfying, I know, but this is usually how I deal with the problem.

**Reply by Alomgir-Bhuiyan** [reply]
I don't know if someone will call it a 'FIX' . Its 2026 now
